arch/x86/chunkset_avx.c

```diff
@@ -45,6 +83,51 @@ static inline void storechunk(uint8_t *out, chunk_t *chunk) {
     _mm256_storeu_si256((__m256i *)out, *chunk);
 }
 
+static inline chunk_t GET_CHUNK_MAG(uint8_t *buf, uint32_t *chunk_rem, uint32_t dist) {
+    lut_rem_pair lut_rem = perm_idx_lut[dist - 3];
+    __m256i ret_vec;
+    /* While technically we only need to read 4 or 8 bytes into this vector register for a lot of cases, GCC is compiling this
+     * to a shared load for all branches, preferring the simpler code.  Given that the buf value isn't in GPRs to begin with,
+     * the 256 bit load is _probably_ just as inexpensive */
+    *chunk_rem = lut_rem.remval;
+
+#ifdef Z_MEMORY_SANITIZER
+    /* See note in chunkset_sse4.c for why this is ok */
```

Comment:
I'm not sure this comment is useful, people usually don't view two files in parallel.

Reply:
I'd prefer not to verbatim copy and paste that lengthy comment.  If the user really has questions as to whether that unpoison call is right, I don't think having them open one more file is the end of the world.